### ISSUE TITLE
WCAG 2.2 Violation - There are links with matching accessible names but different URLs present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#10]
### Issue
There are links with matching accessible names that go to different URLs (different locations), where none of the following exceptions are true:

- The links have additional, distinguishing context by being inline elements inside a `<p>` element with context provided by adjacent text
- The links are by being labelled by `<th>` elements (where the link is inside a `<td>` element, within a `<table>` element)
- The links are receiving programmatically determined link context (via `aria-label` or `aria-labelledby`) (where all the visible text of the link is included as part of the name, along with further, distinguishing context)

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG22/#link-purpose-in-context)

### User Impact

Some users - such as screen reader users, or braille display users - will be unable to distinguish between the different links, or navigate to the specific link(s) they're looking for using assistive technology features (such as screen readers' "Links" views, etc.).

#### Further information:

1. [The "Intent" section of "Understanding Success Criterion 2.4.4: Link Purpose (In Context) | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html#intent)

### Instances

- #30 

#### Using your browser's developer tools:

1. Inspect the link elements using your browser's developer tools.
2. Notice that the links have different URLs, but identical link text (with no alternative mechanisms in use to provide the links with different accessible names from one another, such as `.visually-hidden` text, different `aria-label` values, or different `aria-labelledby` values).

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader and use its "Links" browsing feature.
2. Notice that the links are listed with identical text to one another (despite going to different URLs).

### How To Fix The Issue

Update the links to be distinguishable using one of the following techniques (note: do not combine these techniques together. They are mutually exclusive from one another):

1. (Preferred approach) Update each link's visible text, making each one unique and distinguishable from each other link.
2. Keep the link's visible text as-is, but add additional context within the link using `.visually-hidden` text (ensuring that any necessary white space and/or punctuation is included along with the `.visually-hidden` text, so that the link's final accessible name has the correct spacing and punctuation between words).
3. Keep the link's visible text as-is, but add the `aria-label` attribute to it (ensuring that _all_ of the visible text is included within the `aria-label` value, in addition to further, clarifying text. Also: ensuring that each link's `aria-label` value is unique and distinguishable from the other links in question).

#### Further information:

**If using technique 2 (`.visually-hidden` text):**
1. ["How-to: Hide content - The A11Y Project"](https://www.a11yproject.com/posts/how-to-hide-content/)

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
